Tests/PHPCI/Command/CreateBuildCommandTest.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * PHPCI - Continuous Integration for PHP
+ *
+ * @copyright    Copyright 2014, Block 8 Limited.
+ * @license        https://github.com/Block8/PHPCI/blob/master/LICENSE.md
+ * @link            http://www.phptesting.org/
+ */
+
+namespace Tests\PHPCI\Command;
+
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Tester\CommandTester;
+
+class CreateBuildCommandTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @var \PHPCI\Command\CreateAdminCommand|\PHPUnit_Framework_MockObject_MockObject
+     */
+    protected $command;
+
+    /**
+     * @var \Symfony\Component\Console\Application|\PHPUnit_Framework_MockObject_MockObject
+     */
+    protected $application;
+
+    public function setup()
+    {
+        parent::setup();
+
+        $projectStoreMock = $this->getMockBuilder('PHPCI\\Store\\ProjectStore')
+            ->getMock();
+        $projectStoreMock->expects($this->once())
+            ->method('getById')
+            ->with($this->equalTo(1))
+            ->willReturn($this->getMock('PHPCI\\Model\\Project'));
+
+        $buildServiceMock = $this->getMockBuilder('PHPCI\\Service\\BuildService')
+            ->disableOriginalConstructor()
+            ->getMock();
+
+        $this->command = $this->getMockBuilder('PHPCI\\Command\\CreateBuildCommand')
+            ->setConstructorArgs([$projectStoreMock, $buildServiceMock])
```

Comment:
Use ```array()``` instead of ```[]```. PHPCI support PHP5.3 so we can't use the short array syntax definition.